v0.5/recommendation/README.md

```diff
@@ -0,0 +1,262 @@
+# MLPerf Inference Benchmarks for Recommendation Task
+
+This is the reference implementation for MLPerf Inference benchmarks.
+
+### Supported Models
```

Comment:
It would be useful to know the sizes of the pretrained models before downloading them.

Reply:
I have added them to the README.